challenges/arrayShift/array-shift.js

```diff
@@ -0,0 +1,7 @@
+function insertShiftArray(arr, val) {
+  const middle = arr.length / 2;
+  const middleIndex = Math.ceil(middle);
+  arr.splice(middleIndex, 0, val);
```

Comment:
I would like to see this done in a way that isn't using .splice(), I would suggest using a loop here to write the logic